### ISSUE TITLE
Improve GameServerApp for flexible and faster development

### ISSUE THREAD

**Issue by MatthewFrench** [LeagueSandbox/GameServer#673]
The idea is to make development faster and more involved with team members.

- [x] Create GameServerConsole for only the console version(current).
**Create a UI for Game Server App that includes**
- [x] All JSON options as live UI options 
    - [ ] (If server is running, they modify server behavior)
- [x] Add toggle-able logging level display
- [x] Server should be start and stoppable without closing the app
- [x] Should have option at auto-start server on launch and auto-start client on server ready
- [ ] **Add experimental decentralized  developer match making and chat**
    - [ ] Add match making so you can be in your game or join another game from a connected client
    - [ ] Add chat so everyone can talk to each other
    - [ ] Constantly update the connected list and try to expand the network of clients by asking for IPs

**Comment by MythicManiac** [comment]
It probably shouldn't be P2P because then we'd need a server to do the initial matching, but just decentralized (so self hosted or whatever)

**Comment by MatthewFrench** [comment]
I didn't realize P2P required a central match making server.
I was just going to make it query all connected peers for lobby information.

**Comment by MythicManiac** [comment]
Yeah so I think you had the right concept but wrong terminology used, just remove P2P from the description and you're 👌
